src/components/pages/PageBase.js

```diff
@@ -26,12 +28,19 @@ function PageBase({ children, ...props }) {
     history.replace('/');
   };
 
+  useEffect(() => {
+    if (localStorage.getItem(USER)) {
```

Comment:
Ah I see. If the backend were to return the correct user dict result the first time, we wouldn't have to run an AJAX query below. That should be an easy fix on the backend at some point.